packages/wallet/api/test/test-lib-wallet.js

```diff
@@ -1193,6 +1193,166 @@ test('lib-wallet addOffer for autoswap swap', async (/** @type {LibWalletTestCon
   );
 });
 
+test('lib-wallet performAction suggestIssuer', async (/** @type {LibWalletTestContext} */ t) => {
```

Comment:
context should be typed on `test` binding so they don't have to be repeated in each call. example: https://github.com/Agoric/agoric-sdk/blob/936f9af005eab2333f6460eaeb79a696c60646ce/packages/inter-protocol/test/test-gov-collateral.js#L35-L37

Reply:
Done